for $x at $p in fn:error()
return true()
